packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testNotLeftFlanking.st

```diff
@@ -0,0 +1,12 @@
+tests - state
+testNotLeftFlanking
```

Comment:
Idea: this could be decomposed into `testIsNotLeftFlankingAtEndOfLine`, `testIsNotLeftFlankingWhenFollowedWithWhitespace`, ...